import_logs.py

```diff
@@ -483,6 +486,10 @@ def get(self, key):
     r'.*:\ (?P<ip>[\w*.]+).*\[(?P<date>.*)\].*\ (?P<status>\b\d{3}\b)\ (?P<length>\d+)\ -.*\"(?P<method>\S+)\ (?P<path>\S+).*'
 )
 
+_GANDI_SIMPLE_HOSTING_FORMAT = (
+    r'(?P<host>[0-9a-zA-Z-_.]+)\s+(?P<ip>[a-zA-Z0-9.]+)\s+\S+\s+(?P<userid>\S+)\s+\[(?P<date>.+?)\s+(?P<timezone>.+?)\]\s+\((?P<generation_time_secs>[0-9a-zA-Z\s]*)\)\s+"(?P<method>[A-Z]+)\s+(?P<path>[^\s]+)\s+([^\s]+)"\s+(?P<status>[0-9]+)\s+(?P<length>[^\s]+)\s+"(?P<referrer>[^\s]+)"\s+"(?P<user_agent>[^"]+)"'
+)
```

Comment:
One more change than it should be good to merge: the `[^\s]+` parts can be replaced with just `\S+`, correct?

Reply:
It is much the same thing :
`\s`: any whitespace characters
`\S`: any non-whitespace characters

So, `[^\s]` (all characters except any whitespace characters) is equal to `\S` (any non-whitespace characters).

But if you really want it, I change it :)

Reply:
Hi @tups can you please change it to keep it consistent with the other regexes? Thanks!

Reply:
It is done